contrib/packaging/rpm/finch-buildkit.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
```

Comment:
are the service launch etc are tested somewhere else already?

Reply:
Currently when installing Finch, all of these services are launched automatically by the `%post` scripts in the RPM spec. I'm going to switch to using [RPM systemd scriptlet macros](https://docs.fedoraproject.org/en-US/packaging-guidelines/Scriptlets/#_systemd), so we will now have to manually start finch.service, which will also start all of its related services (currently, buildkit and soci)